starter-code/scripts/models/article.js

```diff
@@ -1,101 +1,126 @@
 'use strict';
 
 (function(module) {
```

Comment:
Nice work wrapping your file in an IIFE to only expose an "interface" for global access!